Makefile

```
@@ -138,16 +138,19 @@ test-db:
 	for pkg in $$(go list ./test/binding/...); do \
-	  HUB_BASE_URL="$(HUB_BASE_URL)" USER="admin" PASSWORD="admin" go test -count=1 -v -failfast "$$pkg" || exit 1; \
+	  HUB_BASE_URL="$(HUB_BASE_URL)" go test -count=1 -v -failfast "$$pkg" || exit 1; \
+	done
+test-auth:
+	for pkg in $$(go list ./test/auth/...); do \
+	  HUB_BASE_URL="$(HUB_BASE_URL)" go test -count=1 -v -failfast "$$pkg" || exit 1; \
 	done
 
 # Run Hub test suite.
-test-all: test-unit test-api
+test-all: test-unit test-binding test-auth
```

checkmake (0.3.2)

[warning] Target "test-all" should be declared PHONY.

(phonydeclared)